### PR TITLE
How to experiment with Election Phragmen pallet

### PR DESCRIPTION
**Pull request by kelvinwelter**
Thanks for submitting this tutorial for Figment Learn!

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://www.notion.so/figmentnetworks/Figment-Learn-Contributor-Guidelines-b08691366e144d33a5636d9276a44ec8) and reviewed the [Sample Tutorial Structure](https://www.notion.so/figmentnetworks/Sample-Tutorial-Structure-667ac2aad36f4f94a3ebaca053180b2d). 

Please confirm the following before submitting your tutorial for review - 

- [X] My tutorial follows the [Tutorial Contribution Guidelines](https://www.notion.so/figmentnetworks/Figment-Learn-Contributor-Guidelines-b08691366e144d33a5636d9276a44ec8).

In particular:

- [X] The title of the Pull Request matches the title of my tutorial.
- [X] I've followed the [tutorial template](https://www.notion.so/figmentnetworks/Sample-Tutorial-Structure-667ac2aad36f4f94a3ebaca053180b2d) structure.
- [X] I've [proofread the tutorial](https://grammarly.com) for spelling and grammar mistakes.
- [X] I've run a linter & prettifier to format my code blocks.
- [X] I've thoroughly explained the code, with inline code comments and text explanations.
- [X] I've run the tutorial to test it and the code works.

**NOTE**: We cannot review your Pull Request if it does not meet the standard outlined above.

### CODE REVIEW

**Comment by yash-sharma1** [comment]
Thanks for the submitting the PR @kelvinwelter I'm glad to see this tutorial coming from you. I'm going to start the review on this now and will let you know if any improvmenets are required. :)

File: polkadot/how-to-experiment-with-election-phragmen-pallet.md [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+<Election accountPair={accountPair} />
+
+<Grid.Row>
+    <TemplateModule accountPair={accountPair} />
+</Grid.Row>
+```
+
+Finally, **you are ready to try the Elections Phragmen pallet!** Go ahead and run tests, create a poll, elect members, run for an election, and so on.
+
+![Election component preview](https://raw.githubusercontent.com/figment-networks/learn-tutorials/master/assets/electionComponentPreview.png?raw=true)
```
Comment (yash-sharma1) [inline polkadot/how-to-experiment-with-election-phragmen-pallet.md:542]: Can you please add a gif/video of working election dApp running on localhost?

**Review by yash-sharma1 — CHANGES_REQUESTED** [review]

**Comment by kelvinwelter** [comment]
Hey @yash-sharma1, I have made the changes you proposed, can you take a look again? Thanks :)

**Review by yash-sharma1 — APPROVED** [review]

**Comment by yash-sharma1** [comment]
Thanks, @kelvinwelter for making all these changes. I'm requesting another review from @pyr0gan before we merge it. Rogan will take review and if all is good we'll merge it.

**Comment by kelvinwelter** [comment]
Hey @pyr0gan, thanks for your feedback! I fixed the details you pointed out, can you take a look again? 

Just one comment about the file created being `Election.js` and not `Election.jsx`: In the Substrate front-end template, which I use here in the tutorial, all ReactJS files end with `.js` and so I decided to do the same with the new file created during the tutorial. This decision was made to try not to confuse the reader of the tutorial. If you prefer, I can make the change to `Election.jsx`, it's up to you :)

**Review by pyr0gan — APPROVED** [review]

**Pull request merged**